### PR TITLE
Add kubevirt hypershift csi periodic jobs 4.14 and 4.15

### PR DESCRIPTION
**Pull request by awels**
Add new periodic jobs for kubevirt csi running on hypershift

### CODE REVIEW

**Comment by awels** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-csi-conformance

File: ci-operator/step-registry/hypershift/kubevirt/run-csi-e2e/hypershift-kubevirt-run-csi-e2e-chain.yaml [hunk 1]
```diff
@@ -20,7 +20,19 @@ chain:
...
         -o "${ARTIFACT_DIR}/e2e.log" \
-        --junit-dir "${ARTIFACT_DIR}/junit"
+        --junit-dir "${ARTIFACT_DIR}/junit" 2>&1 | tee /tmp/openshift-csi-tests.log
+
+      csi_exit_code=${PIPESTATUS[0]}
+
+      # If this line is shown in the openshift-tests output, all of the conformance test cases passed,
+      # but the monitor test has failed. Currently we're ignoring MonitorTest failures until we resolve:
+      # https://issues.redhat.com/browse/CNV-33717
+      if [[ -n "${SKIP_MONITOR_TEST}" ]] && [[ $(grep "failed due to a MonitorTest failure" /tmp/openshift-csi-tests.log) ]];
```
Comment (davidvossel) [inline ci-operator/step-registry/hypershift/kubevirt/run-csi-e2e/hypershift-kubevirt-run-csi-e2e-chain.yaml:30]: you'll need to add the SKIP_MONITOR_TEST env var and set it within the `hypershift-kubevirt-csi-e2e` workflow as well. 

Just do a grep of how `SKIP_MONITOR_TEST` is used for the normal conformance test workflow to see how to copy the same behavior.

File: ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.15-periodics.yaml [hunk 2]
```diff
@@ -699,3 +699,84 @@ periodics:
...
+    org: openshift
+    repo: hypershift
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/variant: periodics
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-csi-conformance
```
Comment (davidvossel) [inline ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.15-periodics.yaml:719]: right below this line, can you add 
```
  reporter_config:
    slack:
      channel: '#hypershift-kubevirt-platform'
      job_states_to_report:
      - failure
      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs> :volcano:'
```

File: ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.14-periodics.yaml [hunk 3]
```diff
@@ -692,6 +692,87 @@ periodics:
...
+    org: openshift
+    repo: hypershift
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/variant: periodics
+    ci.openshift.io/generator: prowgen
+    job-release: "4.14"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-csi-conformance
```
Comment (davidvossel) [inline ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.14-periodics.yaml:712]: right below this line, can you add, that sends failures to our slack channel.
```
  reporter_config:
    slack:
      channel: '#hypershift-kubevirt-platform'
      job_states_to_report:
      - failure
      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs> :volcano:'
```

**Comment by awels** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-csi-conformance

**Comment by awels** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-csi-conformance

**Comment by awels** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-csi-conformance

**Review by davidvossel — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**